src/main/resources/vm/script/datamanagement/downloadFunctions.vm

```diff
@@ -108,6 +108,11 @@ function downloadGirderFile {
     local fileId=`echo $URI | sed -r 's/^.*[?&]fileid=([^&]*)(&.*)?$/\1/i'`
     local token=`echo $URI | sed -r 's/^.*[?&]token=([^&]*)(&.*)?$/\1/i'`
 
+    if [ ! `which girder-client` ]
+    then
+        pip install --user girder-client
+    fi
+
     COMMLINE="girder-client --api-url ${apiUrl} --token ${token} download --parent-type file ${fileId} ./${fileName}"
     echo "downloadGirderFile, command line is ${COMMLINE}"
     ${COMMLINE}
```

Comment:
Not directly related, but the script should exit if it fails

Reply:
The exit in case of failure is already done by the code calling downloadGirderFile.  It is followed by a call to validateDownload.

Reply:
That's right, my bad.